//请求接受任务
package protoFile.quest.TaskNpcAcceptTaskRequest1412;

//请求接受任务
message TaskNpcAcceptTaskRequest{ 
    required int32 id		= 1;			//角色id
	required int32 task_id	= 2;			//任务id
	optional int32 npc_id	= 3;			//npc id
}

message TaskNpcAcceptTaskResponse{
	required bool	result				= 1;			//是否成功
	required int32	task_id				= 2;			//任务id
	optional string message				= 3;			//错误说明
}

